zerver/lib/email_mirror.py

```diff
@@ -406,3 +409,33 @@ def mirror_email_message(data: Dict[str, str]) -> Dict[str, str]:
         }
     )
     return {"status": "success"}
+
+# Email mirror rate limiter code:
+
+class RateLimitedRealmMirror(RateLimitedObject):
+    def __init__(self, realm: Realm) -> None:
+        self.realm = realm
+
+    def key_fragment(self) -> str:
+        return "{}:{}".format(type(self.realm), self.realm.id)
```

Comment:
I think we should have `emailmirror` in this key as well, since this rate-limit is just scoped to that subsystem.